src/sentry/integrations/mixins/issues.py

```diff
@@ -99,9 +121,15 @@ def get_group_link(self, group, **kwargs):
 
     def get_group_description(self, group, event, **kwargs):
         output = self.get_group_link(group, **kwargs)
-        body = self.get_group_body(group, event)
-        if body:
-            output.extend(["", "```", body, "```"])
+        if isinstance(event, GroupEvent) and event.occurrence is not None:
+            body = ""
+            if group.issue_category == GroupCategory.FEEDBACK:
```

Comment:
Do we want to output nothing if the category != Feedback?

Reply:
if the category isn't feedback it'll return the default text before the table, which is what it always has returned

Reply:
<img width="445" alt="SCR-20240812-nlxr" src="https://github.com/user-attachments/assets/66407af6-f398-4463-bdb8-2fe785a85954">

Reply:
Is there no need to call `get_group_body` for this case?

Reply:
<img width="500" alt="SCR-20240812-npuf" src="https://github.com/user-attachments/assets/bea92782-e5c2-4ed9-bd9d-0c35c48be6a8">

it looks like currently for feedback, `get_group_body` just returns nothing. probably this line doesn't execute because `event.interfaces.values()` doesn't exist

https://github.com/getsentry/sentry/blob/dfcfaa7efa692bb161df9157ece589f98dacada1/src/sentry/integrations/mixins/issues.py#L78

so not calling it won't have an effect on feedback descriptions

Reply:
I meant for non-Feedbacks, so if 
`isinstance(event, GroupEvent) and event.occurrence is not None and group.issue_category != GroupCategory.Feedback`

Reply:
> if the category isn't feedback it'll return the default text before the table

What/where is the default text? Sorry if I'm kinda behind here

Reply:
ahh i see what you mean -- i'll add the feedback category check into the main if statement, so if the feedback isn't issues it'll still default to the `else` case